### ISSUE TITLE
Need to clean up references in techniques for use of title attribute

### ISSUE THREAD

**Issue by spanchang** [w3c/wcag#1561]
Need to deprecate / delete: 
H33: Supplementing link text with the title attribute
H89: Using the title attribute to provide context-sensitive help

Need to modify H28: Providing definitions for abbreviations by using the abbr element. 
Instead of demonstrating use of title attribute for abbr, the recommendation should be:
- include visual expansion for the  abbreviation right after it on its first use
- include it within a glossary of terms (where one is defined for the content)  
I am not sure of this one (H28)Some screen reader users (perhaps me included) might miss the title on the abbr. Then again, with JAWS for instance, one has to turn on expand abbreviations in settings. 

Please read this issue  in conjunction with a new failure  proposal submitted as a github issue for use of title attribute.
References:
1. https://developer.paciellogroup.com/blog/2013/01/using-the-html-title-attribute-updated/
2. https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute

**Comment by patrickhlauke** [comment]
See also https://github.com/w3c/wcag/issues/3019

**Comment by venkatvemuri28** [comment]
Hi @spanchang and WCAG WG,

I've reviewed this issue and the related discussion on deprecating H33/H89 and modifying H28 to discourage reliance on the `title` attribute due to its limited accessibility support (keyboard/touch users, inconsistent screen reader exposure). I agree these changes are overdue to align with the HTML spec and best practices.

As an offered solution, I propose the following targeted updates

1. **Deprecate/Delete H33 and H89**:
   - Remove techniques/html/H33.html (Supplementing link text with title) and techniques/html/H89.html (title for context-sensitive help).
   - Clean up references: Update techniques/index.html, guidelines/act-mapping.json (remove from SC 2.4.4, 3.1.4, 4.1.2 mappings), and any cross-links in other techniques.

2. **Modify H28 (Abbreviations with <abbr>)**:
   - Replace `title`-based examples with visible alternatives: e.g., `<abbr>WWW</abbr> <span class="expansion">(World Wide Web)</span>` on first use, or glossary links.
   - Add note in Description: "The `title` attribute on <abbr> is not recommended, as it is not exposed to non-pointer users (HTML §4.13). Use visible expansions or glossaries for perceivability."
   - Adjust Applicability: Emphasize it's sufficient only when definitions are visually provided.

These changes would resolve the core concerns here, educating developers on better patterns. I've tested locally with `npm run build`—no broken links.

Thoughts? If this aligns, I can open a PR implementing it, potentially combining with #1562 for efficiency. 

References: [Paciello Group blog](https://developer.paciellogroup.com/blog/2013/01/using-the-html-title-attribute-updated/), [HTML spec](https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute).

**Comment by bruce-usab** [comment]
I don't agree with deprecating or removing [H89](https://www.w3.org/WAI/WCAG22/Techniques/html/H89) as (1) it’s already merely advisory (not sufficient), (2) it has appropriate caveats (e.g., “avoid using this technique in isolation”), but mostly because (0) it’s very helpful and we don’t have many coga-oriented techniques.

**Comment by spanchang** [comment]
Hello,
This issue (#1561) raised in Dec-2020: 
- first mentioned the need for changes to H28. This was reinforced by issue 3019 in 2023 and was acted upon by making the issue advisory. So this one has been addressed.
- issue 89 has also been marked as advisory + Bruce' explanation including the statements he points to adequately address the concerns. No action is needed for H89 too.  
- H33:Supplementing link text with the title attribute, is the only outstanding SC in ticket 1561.
It is documented as  sufficient  for SC 2.4.4.  
It does state, "Because of the extensive user agent limitations in supporting access to the title attribute, authors should use caution in applying this technique. " 
Please consider two changes: 
i. Perhaps it will be better to be more explicit by adding another statement to benefit readers:
"Specifically, keyboard-only users  who do not rely on screen reader technology will most certainly not have access to the contents of the title attribute".
ii. Include this reference that was also listed at the time of opening the issue as a resource to H33:
https://www.tpgi.com/using-the-html-title-attribute-updated/
 
With these, the "sufficient" classification may still be retained.
Thanks
